dt-people-groups/people-groups-base.php

```diff
@@ -73,8 +73,24 @@ public function dt_details_additional_section( $section, $post_type ) {
     }
 
     public function dt_nav_filter( $navigation_array ) {
+        $is_hidden = ! get_option( Disciple_Tools_People_Groups::$option_key_settings_display_tab );
+
         if ( isset( $navigation_array['main'], $navigation_array['main'][ $this->post_type ] ) ) {
-            $navigation_array['main'][ $this->post_type ]['hidden'] = ! get_option( Disciple_Tools_People_Groups::$option_key_settings_display_tab );
+            $navigation_array['main'][ $this->post_type ]['hidden'] = $is_hidden;
+        }
+
+        if ( isset( $navigation_array['admin'], $navigation_array['admin']['add_new'] ) ) {
+            $updated_submenus = [];
+            foreach ( $navigation_array['admin']['add_new']['submenu'] ?? [] as $submenu ) {
```

Comment:
can we add the `&` before $submenu. so we don't need the temp array?

Reply:
@corsacca Funny, as this was my first approach, to update by reference (&); but for some reason, it didn't take and did not seem happy with the nested submenus! Not to worry, as I'll circle back and try again... No doubt, it's me missing something!! 😉